ooiui/core/routes/science.py

```diff
@@ -77,3 +77,17 @@ def flush():
 def root():
     return render_template('science/index.html', erddap_url=ERDDAP_URL)
 
+@app.route('/api/array')
+def array_proxy():
+    response = requests.get(SERVICES_URL + '/arrays', params=request.args)
+    return response.text, response.status_code
+
+@app.route('/api/platform_deployment')
+def platform_deployment_proxy():
+    response = requests.get(SERVICES_URL + '/platform_deployments', params=request.args)
+    return response.text, response.status_code
+
+@app.route('/api/instrument_deployment')
+def instrument_deployment_proxy():
+    response = requests.get(SERVICES_URL + '/instrument_deployments', params=request.args)
+    return response.text, response.status_code
```

Comment:
We should probably develop an annotation or a wrapper for these sorts of proxies.